demos/text_detection_demo/src/text_detection.cpp

```diff
@@ -98,7 +98,36 @@ std::vector<cv::RotatedRect> maskToBoxes(const cv::Mat &mask, float min_area, fl
     }
 
     return bboxes;
-  }
+}
+
+std::vector<cv::RotatedRect> coordToBoxes(const float* coords,
+                                          const size_t &size_data,
+                                          float min_area, float min_height,
+                                          const cv::Size &input_shape,
+                                          const cv::Size &image_size) {
+    std::vector<cv::RotatedRect> bboxes;
+    int max_bbox_idx = size_data / 5;
```

Comment:
```suggestion
    int num_boxes = size_data / 5;
```

Seems like this actually isn't a valid bounding box index. It's the number of boxes.